crates/router/src/services/kafka.rs

```diff
@@ -268,26 +272,29 @@ impl KafkaProducer {
         old_attempt: Option<PaymentAttempt>,
     ) -> MQResult<()> {
         if let Some(negative_event) = old_attempt {
-            self.log_event(&KafkaEvent::old(&KafkaPaymentAttempt::from_storage(
-                &negative_event,
-            )))
+            self.log_event(&KafkaEvent::old(
+                &KafkaPaymentAttempt::from_storage(&negative_event),
+                TenantID("default".to_string()),
```

Comment:
can you use the tenant_id stored in the kafkastore instead?

Reply:
Yes I have used that TenantID here only ... I have imported that at the top.

Reply:
I mean that can you add a parameter in the log_payment_attempt function to accept a tenant_id and use that instead.

Reply:
Yes done that!